app/controllers/static_controller.rb

```diff
@@ -0,0 +1,23 @@
+class StaticController < ApplicationController
+  include Decidim::LocaleSwitcher
+
+  layout "decidim/application"
+
+  helper_method :current_organization, :decidim_page_title
+
+  def accountability
+  end
+
+  def accountability_sections
+  end
+
+  private
+
+  def current_organization
+    @current_organization ||= request.env["decidim.current_organization"]
```

Comment:
You could just include:

```ruby
include Decidim::NeedsOrganization
```

And this will both ensure you won't access this without an organization and include the `current_organization` as a helper.